test/erc20/myErc20Contract.test.js

```diff
@@ -0,0 +1,148 @@
+const { expect, assert } = require('chai');
+
+// Import utilities from Test Helpers
+const { BN, expectEvent, expectRevert, constants } = require('@openzeppelin/test-helpers');
+const { web3 } = require('@openzeppelin/test-helpers/src/setup');
+
+// Load compiled artifacts
+const erc20 = artifacts.require('MYERC20');
+
+let accounts;
+
+// Start test block
+contract('MYERC20', () => {
+
+  const NAME = 'MyToken';
+  const SYMBOL = 'My';
+
+  beforeEach(async ()  => {
+    accounts = await web3.eth.getAccounts()
+  })
+
+  it('tests that contract has 18 decimals', async () =>  {
+    const instance = await erc20.new(NAME, SYMBOL)
```

Comment:
you can see that you're doing the same thing in every case. Move this call to `beforeEach`